CMakeLists.txt

```diff
@@ -228,6 +228,15 @@ if ( NOT ${WRF_CASE} IN_LIST WRF_CASE_OPTIONS )
   message( FATAL_ERROR "WRF Case option not recognized : ${WRF_CASE}" )
 endif()
 
+# handle Hydro options
+if ( ${WRF_HYDRO} )
```

Comment:
Where is `WRF_HYDRO` being set?

Reply:
Removed this section after switch to `ENABLE_HYDRO`